utils/g2c_index.c

```diff
@@ -43,6 +45,9 @@ main(int argc, char **argv)
         case 'v':
             verbose = 1;
             break;
+        case 'l':
```

Comment:
I was going to ask whether documentation might need updating, but for the utilities for example I don't see any documentation. Would it make sense to add an issue to add utility documentation?

Reply:
Yes, that's a good point. I will do that. However, for now, the g2c utilities are considered experimental. Really they exist so that I can test the C code the same way we test the fortran code, and make sure everybody is coming up with the same answers.

Using degrib2 output, for example, is a great way to ensure that the two libraries are reading a file the same way.